### DISCUSSION TITLE
查询 root.a.b 成功 查询 root.a.c失败

### DISCUSSION THREAD

**Discussion by StrongManLi** [apache/iotdb#10737] [Q&A] [answered]
![image](https://github.com/apache/iotdb/assets/58460020/02a3dff7-67ef-4cc2-a514-4bbbf0629b1f)
root.a.b成功
root.a.c失败
root.a.*失败

**Comment by StrongManLi** [comment, marked as answer]
问题的原因找到了，路径当中包含了 null，删了就好了

**Reply by jixuan1989** [reply]
特殊字符/关键词，需要用``符号

**Comment by StrongManLi** [comment]
谢谢



---原始邮件---
发件人: "Xiangdong ***@***.***&gt;
发送时间: 2023年8月1日(周二) 晚上10:06
收件人: ***@***.***&gt;;
抄送: "StrongMan ***@***.******@***.***&gt;;
主题: Re: [apache/iotdb] 查询 root.a.b 成功 查询 root.a.c失败 (Discussion #10737)




 
特殊字符/关键词，需要用``符号
